### PR TITLE
Add "--narrow" option for more details in case of frameskips present

### PR DESCRIPTION
**Pull request by Hermann-SW**
Please see arguments for "--narrow" option in this posting:
https://forums.raspberrypi.com/viewtopic.php?p=2171665#p2171665

"--narrow" allows for avoiding frameskips dominate y-axis resolution:

### CODE REVIEW

File: utils/timestamp.py [hunk 1]
```diff
@@ -67,6 +70,7 @@ def plot_pts(diffs, avg, title):
     parser.add_argument('filename', help='PTS file generated from rpicam-vid (with a .txt or .pts extension)'
                                          ' or an avi/mkv/mp4 container file', type=str)
     parser.add_argument('--plot', help='Plot timestamp graph', action='store_true')
+    parser.add_argument('--narrow', help='Frameskips outside view', action='store_true')
```
Comment (naushir) [inline utils/timestamp.py:73]: Could this description be changed to something like "Narrow the y-axis by hiding outliers" or similar please?

**Review by naushir — COMMENTED** [review]
Change looks reasonable.  One additional thing, could you update the commit message to follow the format of the other commit messages, i.e. have a header like "utils: timestamp: Add --narrow option", a small description, and a "Signed-off-by:" tag.  Have a look at the git log to see some examples.

**Comment by Hermann-SW** [comment]
@naushir I did the changes you requested, and followed instructions regarding <kbd>git commit --amend</kbd>:
https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/changing-a-commit-message

I got into trouble with merge conflicts, but finally got problems resolved.
The new hash shows both requested changes, to commit message and line 73 of timestamp.py:
https://github.com/Hermann-SW/rpicam-apps/commit/0fe3a4209c96fa62e23d22a45e5c0fbb0d55af14

File: utils/timestamp.py (hunk 1 above)
Comment (Hermann-SW) [reply utils/timestamp.py]: @naushir I did the changes you requested, and followed instructions regarding <kbd>git commit --amend</kbd>:
https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/changing-a-commit-message

I got into trouble with merge conflicts, but finally got problems resolved.
The new hash shows both requested changes, to commit message and line 73 of timestamp.py:
https://github.com/Hermann-SW/rpicam-apps/commit/0fe3a4209c96fa62e23d22a45e5c0fbb0d55af14

**Comment by naushir** [comment]
Thanks for the updates.  I'll squash these changes and merge them manually now.

**Pull request closed without merging**